wrangler-core/src/main/java/io/cdap/wrangler/registry/UserDirectiveRegistry.java

```diff
@@ -129,8 +129,8 @@ public DirectiveInfo get(String namespace, String name) throws DirectiveLoadExce
       }
       if (directive == null) {
         throw new DirectiveLoadException(
-          String.format("10-5 - Unable to load the user defined directive '%s'. " +
-                          "Please check if the artifact containing UDD is still present.", name)
+          String.format("Directive '%s' not found. Check if directive is spelled correctly or if directive is " +
```

Comment:
nit: Check if the directive is spelled corrected. If the directive is user defined, make sure the artifact has been uploaded.